lib/field.ts

```diff
@@ -57,11 +57,10 @@ export default class Field {
       locale = undefined
     }
 
-    return this._getFieldLocale(locale).onIsDisabledChanged(handler)
+    return this.getFieldLocale(locale).onIsDisabledChanged(handler)
   }
 
-  private _getFieldLocale(locale) {
-    locale = locale || this._defaultLocale
+  getFieldLocale(locale = this._defaultLocale) {
```

Comment:
After reading the documentation PR you put up and looking at this method again, I'm a bit confused as a user. Why does a method called `getFieldLocale` have an optional argument to specify a locale?

Reply:
Yeah, the naming is not great. I think the distinction between a `field` and what we have called a `fieldLocale` is probably quite hard to understand for users full stop. I think it's made extra confusing by the fact that we don't really express the difference between them in the docs.

I'm really struggling to think of a good name for this - given the names of our types it should be something like `getFieldFromEntryField` - but that is kind of baffling